src/MoBi.Presentation/Presenter/EditMoleculeBuilderPresenter.cs

```diff
@@ -54,6 +54,9 @@ internal class EditMoleculeBuilderPresenter : AbstractSubPresenterWithFormula<IE
          _view.UpdateStartAmountDisplay(dimensionRetriever.SelectedDimensionMode == ReactionDimensionMode.AmountBased
             ? AppConstants.Captions.Amount
             : AppConstants.Captions.Concentration);
+
+         //We support container criteria for parameters in molecule builder
+         _editMoleculeParameters.EnableContainerCriteriaSupport();
```

Comment:
We do not want container criteria support by default. Only for molecule parameters for now